lib/rouge/lexers/svelte.rb

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8 -*- #
+# frozen_string_literal: true
+
+module Rouge
+  module Lexers
+    class Svelte < HTML
+      desc 'Svelte single-file components (https://svelte.dev/)'
+      tag 'svelte'
+      filenames '*.svelte'
+
+      mimetypes 'text/x-svelte', 'application/x-svelte'
```

Comment:
Remove extra newline

```suggestion
      mimetypes 'text/x-svelte', 'application/x-svelte'
```